_data/engines.yml

```diff
@@ -1,4 +1,236 @@
+
+- name: Sunda
+  url: http://www.sunda.fi
+  languages: [ [en, fi], [fi, en] ]
+  info_url: https://www.sunda.fi/en/technology-tools.html
+
+- name: Systran
+  url: https://www.systran.net
+  languages: [ [de, en], [de, es], [de, fr], [de, it], [de, ko], [de, pt], [en, ar], [en, bg], [en, bn], [en, ca], [en, cs], [en, cy], [en, da], [en, de], [en, el], [en, es], [en, et], [en, fa], [en, fi], [en, fr], [en, he], [en, hi], [en, hr], [en, hu], [en, id], [en, is], [en, it], [en, ja], [en, ka], [en, ko], [en, lt], [en, lv], [en, ms], [en, nl], [en, no], [en, pl], [en, ps], [en, pt], [en, ro], [en, ru], [en, sk], [en, sl], [en, sr], [en, sv], [en, sw], [en, ta], [en, th], [en, tl], [en, tr], [en, uk], [en, ur], [en, vi], [en, zh-cn], [en, zh-tw], [es, ar], [es, de], [es, en], [es, fr], [es, it], [es, ko], [es, pt], [fr, ar], [fr, de], [fr, el], [fr, en], [fr, es], [fr, it], [fr, nl], [fr, pt], [fr, zh-cn], [fr, zh-tw], [ko, de], [ko, en], [ko, es], [ko, fr], [ko, id], [ko, ja], [ko, ru], [ko, th], [ko, vi], [ko, zh-cn], [ko, zh-tw], [pa, en], [prs, en], [ru, ar], [ru, en], [ru, ko], [so, en], [sq, en], [tg, en] ]
+  info_url: https://www.systransoft.com/download/product-datasheets/systran-languages-datasheet-0120.pdf
+
+- name: Tarjama NMT
+  url: http://www.tarjama.com
+  languages: [ [en, ar], [ar, en] ]
+  info_url: https://github.com/tarjama/translate
+
+- name: Tencent
+  url: https://cloud.tencent.com/product/tmt
+  languages: [ [zh-cn, en], [zh-cn, ja], [zh-cn, ko], [zh-cn, fr], [zh-cn, es], [zh-cn, it], [zh-cn, de], [zh-cn, tr], [zh-cn, ru], [zh-cn, pt], [zh-cn, vi], [zh-cn, id], [zh-cn, th], [zh-cn, ms], [zh-tw, en], [zh-tw, ja], [zh-tw, ko], [zh-tw, fr], [zh-tw, es], [zh-tw, it], [zh-tw, de], [zh-tw, tr], [zh-tw, ru], [zh-tw, pt], [zh-tw, vi], [zh-tw, id], [zh-tw, th], [zh-tw, ms], [en, zh], [en, ja], [en, ko], [en, fr], [en, es], [en, it], [en, de], [en, tr], [en, ru], [en, pt], [en, vi], [en, id], [en, th], [en, ms], [en, ar], [en, hi], [ja, zh], [ja, en], [ja, ko], [ko, zh], [ko, en], [ko, ja], [fr, zh], [fr, en], [fr, es], [fr, it], [fr, de], [fr, tr], [fr, ru], [fr, pt], [es, zh], [es, en], [es, fr], [es, it], [es, de], [es, tr], [es, ru], [es, pt], [it, zh], [it, en], [it, fr], [it, es], [it, de], [it, tr], [it, ru], [it, pt], [de, zh], [de, en], [de, fr], [de, es], [de, it], [de, tr], [de, ru], [de, pt], [tr, zh], [tr, en], [tr, fr], [tr, es], [tr, it], [tr, de], [tr, ru], [tr, pt], [ru, zh], [ru, en], [ru, fr], [ru, es], [ru, it], [ru, de], [ru, tr], [ru, pt], [pt, zh], [pt, en], [pt, fr], [pt, es], [pt, it], [pt, de], [pt, tr], [pt, ru], [vi, zh], [vi, en], [id, zh], [id, en], [th, zh], [th, en], [ms, zh], [ms, en], [ar, en], [hi, en] ]
+  info_url: https://cloud.tencent.com/document/api/551/15619
+
+- name: TextShuttle
+  url: https://www.textshuttle.ai/
+  languages: [ de, en, es, fr, it ]
```

Comment:
This is difficult.  They offer "any" other desired languages.

Reply:
Right, I considered stock engines for this page, as many engines, like KantanMT, offer customizable languages as per their client's request.
I added `custom_languages: null`.

Reply:
Maybe we should put `*`?